flang/test/Lower/MIF/sync_team.f90

```diff
@@ -0,0 +1,25 @@
+! RUN: %flang_fc1 -emit-hlfir -fcoarray %s -o - | FileCheck %s --check-prefixes=COARRAY
+! RUN2: not %flang_fc1 -emit-hlfir %s -o - | FileCheck %s --check-prefixes=NOCOARRAY
+
+program test_sync_team
+  use, intrinsic :: iso_fortran_env, only: team_type
+  implicit none
+  ! NOCOARRAY: Coarrays disabled, use '-fcoarray' to enable.
```

Comment:
Why is this error message different from the rest?

Strictly speaking, `sync team` is a multi-image feature and NOT a coarray feature. As such, the multi-image error message would be more appropriate here.

Reply:
Sorry, that's not the message I meant to print.